### ISSUE TITLE
Decline Button

### ISSUE THREAD

**Issue by waltprt** [nextcloud/terms_of_service#750]
There is only the agree/accept button. Please allow a decline button and one should be logged out Immediately after declining. If this user logs will be available to the administrator would be great otherwise it's ok.

**Comment by cdstg** [comment]
Yes the ability to logout at least is important as if there is no way to logout when the Terms appear
That means there browser is still logged in

As for recording this its not so important for us as I can see their last logged in and also the status in the oc_termsofservice_sigs  so if they are logged in and not in the oc_termsofservice_sigs they have not acceptance
